docs/SCRIPTS.md

```
@@ -168,6 +168,28 @@ node analyze-patterns.mjs --self-test
+
+Folds compensation observations into per-application desired/advertised/actual values and gap aggregates. Sources: `reports/*.md` Machine Summary `advertised_comp` (advertised, source `jd` — historical reports backfill automatically), `data/salary-observations.tsv` (desired/actual, append-only), and `config/profile.yml` `compensation.target_range` (desired default). Fold precedence: highest trust tier wins, then latest date (`actual`: contract > offer-letter > recruiter-verbal > user). Aggregates group by (company, role) and per currency — no FX conversion. Unparseable amounts, orphaned tracker numbers, sample sizes, and staleness are always reported.
+
+```bash
+node salary-gap.mjs             # JSON
+node salary-gap.mjs --summary   # table + data-quality section
+node salary-gap.mjs --self-test
+```
+
+Observation line format (TSV, one per line, `#`-prefixed lines are comments):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)